LondonBritishLibrary/orient/BLorient13278.xml

```diff
@@ -0,0 +1,232 @@
+                                <locus from="90v" to="91r"/>
+                                <title ref="LIT3587Miracle"/>
+                                <incipit xml:lang="gez">ተአምሪሃ፡ ለእግዝእት፡ ቅድስት፡ ድንግል፡ በ፪ማርያም፡ ወላዲተ፡ አምላክ፡ ጸሎታ፡ ወበረከታ፡ <gap reason="ellipsis"/> 
+                                    ተብሕለ፡ ከመ፡ ሀሎ፡ ፩መ<supplied reason="undefined" resp="PRS8999Strelcyn">ነ</supplied>ኮስ፡ ወአልቦቱ፡ ሲሳይ፡ ዘየሴሰይ፡ 
+                                    ወአልባስ፡ ወይትአምና፡ ለእግዝእትነ፡ ማርያም፡ ወላዲተ፡ አምላክ፡ ወይቤላ፡ አዝዚ፡ ሊተ፡ ዘእበልዕ፡ ወዘእለብስ፡ ወዘንተ፡ ብሂሎ፡ ኖመ፤ </incipit>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            <msItem xml:id="ms_i3.3">
+                                <locus from="91r" to="91v"/>
+                                <title ref="LIT3906Miracle"/>
+                                <incipit xml:lang="gez">ተአምሪሃ፡ ለእግዝእትነ፡ ቅድስት፡ ድንግል፡ በ፪ማርያም፡ ወላዲተ፡ አምላክ፡ ጸሎታ፡ ወበረከታ፡ <gap reason="ellipsis"/>
+                                    ወሀሎ፡ ፩መነኮስ፡ ዘይፀውም፡ ፆመ፡ ዚአሀ፡ በንጽሕ፡ ወአል<supplied reason="undefined" resp="PRS8999Strelcyn">ቦ</supplied>ቱ፡ 
+                                    ምግባረ፡ ሠናይ፤ ወበጊዜሃ፡ መጽአ፡ <sic>መላከ፡</sic> ሞት፡ ወወሰደ፡ ነፍሶ፡ ውስተ፡ ሲኦል፡ ወሖረት፡ እግዝእትነ፡ ማርያም፡ ኀበ፡ ወልዳ፡ ወትቤሎ፡ መሐር፡ 
+                                    <sic>ለ</sic>ተ፡ ኦወልድየ፡ ዛቲ፡ ነፍስ፡</incipit>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">92</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>120</height>
+                                        <width>90</width>
+                                    </dimensions>
+                                </extent></supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="12 16"/>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Not very careful hand.</desc>
+                                <date notBefore="1900" notAfter="1978"/>
+                            </handNote>                           
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#1r"/>
+                                <desc><ref type="authFile" corresp="AT1041HolyMen">Miniature of 
```

Comment:
NB every `corresp `= hyperlink
Now you have one hyperlink (`persName`) inside of another (`ref`), the computer cannot process this.
Please do not overlap.
either close the `ref `immediately `<ref type="authFile" corresp="AT1041HolyMen"/> `  or at least before `persName `starts